src/Microsoft.ML.Data/DataLoadSave/Text/TextLoaderSaverCatalog.cs

```diff
@@ -21,7 +21,11 @@ public static class TextLoaderSaverCatalog
         /// <param name="catalog">The <see cref="DataOperationsCatalog"/> catalog.</param>
         /// <param name="columns">Array of columns <see cref="TextLoader.Column"/> defining the schema.</param>
         /// <param name="separatorChar">The character used as separator between data points in a row. By default the tab character is used as separator.</param>
-        /// <param name="hasHeader">Whether the file has a header.</param>
+        /// <param name="hasHeader">Whether the file has a header with feature names. Note: If a TextLoader is created with HasHeader = true but without a
```

Comment:
>HasHeader [](start = 124, length = 9)

Nit: Please match the case of the parameter 'hasHeader' in all instances in the updated docs.

Reply:
Or use <paramref ...> both for `hasHeader` and for `dataSample`.

---
In reply to: [373803799](https://github.com/dotnet/machinelearning/pull/4655#discussion_r373803799) [](ancestors = 373803799)